test/fixtures/correct.js

```diff
@@ -0,0 +1,271 @@
+// Avoid extra `no-unused-vars` violations.
+function noop() {
+  // do nothing
+}
+
+// `array-bracket-spacing`
+noop(['foo', 'bar']);
+
+// `comma-dangle`
+noop({ bar: 'foo', foo: 'bar' });
+
+// `comma-spacing`
+noop(['bar', 'foo']);
+
+// `comma-style`
+noop({ bar: 'foo', foo: 'bar' });
```

Comment:
This is the same as in L10, right? If so, what about merging the comments, instead?